### PR TITLE
Added Twoyo keyboard

### PR DESCRIPTION
**Pull request by rainkeebs**
## Description
Added Twoyo keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/twoyo/readme.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb twoyo -kb default
+    
+To reset the keyboard, hold the top left key while plugging in, or hit the reset button near the USB port
+
+Install example for this keyboard:
+
+    qmk flash -kb twoyo -km default
```
Comment (fauxpark) [inline keyboards/twoyo/readme.md:20]:
```suggestion
    make twoyo:default
    
Flashing example for this keyboard:

    make twoyo:default:flash

To reset the keyboard, hold the top left key while plugging in, or hit the reset button near the USB port
```

File: keyboards/twoyo/rules.mk [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+MOUSEKEY_ENABLE = yes   # Mouse keys
+EXTRAKEY_ENABLE = yes   # Audio control and System control
+CONSOLE_ENABLE = no     # Console for debug
+COMMAND_ENABLE = no     # Commands for debug and configuration
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes       # USB Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no   # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+ENCODER_ENABLE = yes
+SPLIT_KEYBOARD = yes
```
Comment (fauxpark) [inline keyboards/twoyo/rules.mk:20]:
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
ENCODER_ENABLE = yes
SPLIT_KEYBOARD = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
Please also delete the zip file and VIA json, and revert the submodule changes.

**Comment by rainkeebs** [comment]
im gonna go ahead and delete this for now.... only way i know how to fix the stupid chibiOS stuff :D

**Pull request closed without merging**